statsmodels/genmod/families/family.py

```diff
@@ -1432,3 +1432,193 @@ def resid_anscombe(self, endog, mu):
         return ((hyp2f1(-self.alpha * endog) - hyp2f1(-self.alpha * mu) +
                  1.5 * ( endog**(2 / 3.) - mu**(2 / 3.))) /
                 (mu + self.alpha * mu**2)**(1 / 6.))
+
+
+class Tweedie(Family):
+    """
+    Tweedie family.
+
+    Parameters
+    ----------
+    link : a link instance, optional
+        The default link for the Tweedie family is the log link when the
+        link_power is 0. Otherwise, the power link is default.
+        Available links are log, cloglog, identity, nbinom and power.
+        See statsmodels.family.links for more information.
+    var_power : float, optional
+        The variance power.
+    link_power : float, optional
```

Comment:
After an initial browsing through the families and link code:
Your tweedie implementation here follows the same pattern, except that there are no parameterized links yet.
I think this was refactored a long time ago to use only classes instead of instances for links.

I really prefer that we use `link=link.Power(link_power)` in the signature but that needs refactoring of the valid link check